api/notd/manager.py

```diff
@@ -38,105 +40,14 @@
-    {'date': datetime.datetime(2021, 11, 8), 'token': Token(registryAddress='0x495f947276749Ce646f68AC8c248420045cb7b5e', tokenId='37245476930711271386485694793363763377086733366271522319079897520971769184257')},
-    {'date': datetime.datetime(2021, 11, 10), 'token': Token(registryAddress='0x1cf33f4c6c4e6391f4d2b445aa3a36639b77dd68', tokenId='711')},
-    {'date': datetime.datetime(2021, 11, 11), 'token': Token(registryAddress='0xd07dc4262BCDbf85190C01c996b4C06a461d2430', tokenId='685820')},
-    {'date': datetime.datetime(2021, 11, 13), 'token': Token(registryAddress='0x1cf33f4c6c4e6391f4d2b445aa3a36639b77dd68', tokenId='711')},
-    {'date': datetime.datetime(2021, 11, 14), 'token': Token(registryAddress='0x495f947276749Ce646f68AC8c248420045cb7b5e', tokenId='114892502508796003086026794793400110362742126516063867859178063184437467676673')},
-    {'date': datetime.datetime(2021, 11, 16), 'token': Token(registryAddress='0x1cf33f4c6c4e6391f4d2b445aa3a36639b77dd68', tokenId='711')},
-    {'date': datetime.datetime(2021, 11, 17), 'token': Token(registryAddress='0x495f947276749Ce646f68AC8c248420045cb7b5e', tokenId='50483018599523793031646892055201443028457586511343357127301131486835876298753')},
-    {'date': datetime.datetime(2021, 11, 19), 'token': Token(registryAddress='0x1cf33f4c6c4e6391f4d2b445aa3a36639b77dd68', tokenId='711')},
-    {'date': datetime.datetime(2021, 11, 20), 'token': Token(registryAddress='0x495f947276749Ce646f68AC8c248420045cb7b5e', tokenId='48009777219687553353266573207856084490164331626444899155064281984235699437569')},
-    {'date': datetime.datetime(2021, 11, 22), 'token': Token(registryAddress='0x1cf33f4c6c4e6391f4d2b445aa3a36639b77dd68', tokenId='711')},
-    {'date': datetime.datetime(2021, 11, 23), 'token': Token(registryAddress='0x3B3ee1931Dc30C1957379FAc9aba94D1C48a5405', tokenId='88362')},
-    {'date': datetime.datetime(2021, 11, 25), 'token': Token(registryAddress='0x1cf33f4c6c4e6391f4d2b445aa3a36639b77dd68', tokenId='711')},
-    {'date': datetime.datetime(2021, 11, 26), 'token': Token(registryAddress='0x495f947276749Ce646f68AC8c248420045cb7b5e', tokenId='20413448389158464120102864593781297035145279087422163667103007730008737185793')},
-    {'date': datetime.datetime(2021, 11, 28), 'token': Token(registryAddress='0x1cf33f4c6c4e6391f4d2b445aa3a36639b77dd68', tokenId='711')},
-    {'date': datetime.datetime(2021, 11, 29), 'token': Token(registryAddress='0x495f947276749Ce646f68AC8c248420045cb7b5e', tokenId='37735198289038997993427628448167501432332343899935625684167414261625118523393')},
-]
+def load_sponsored_token() -> List[SponsoredToken]:
+    sponsoredList = []
+    with open("notd/sponsored_tokens.json", "r") as sponsoredTokenFile:
+        sponsoredItems = json.load(sponsoredTokenFile)
+        for sponsoredItem in sponsoredItems:
+            sponsoredItem = ast.literal_eval(str(sponsoredItem))
```

Comment:
ah dont do this please - its generally very bad practice. It would be much nicer to have `from_dict` functions on the models. That way this code should look like:
```
    with open("./notd/sponsored_tokens.json", "r") as sponsoredTokensFile:
        sponsoredTokensDicts = json.loads(sponsoredTokensFile.read())
    sponsoredTokens = [SponsoredToken.from_dict(sponsoredTokenDict) for sponsoredTokenDict in sponsoredTokensDicts]
```